<div class='row'>
    <div class='col-md-10 col-md-offset-1'>
        <article>
            <p>The "Similarities" tools allow users to calculate the similarity of receptors based on a custom selection of sequence
            segments. Four tools are found under similarities:</p>
            <section>
                <header><h3>Similarity search with a reference receptor</h3></header>
                <p>Using the similarity search tool is a three step process. The user is first presented with a reference receptor selection
                page.</p>
                <p>Once a reference receptor has been selected, the user is presented with a sequence segment selection page. The user
                can select one or more sequence segments, and/or expand each segment to select the residues within it individually. Residues
                selected individually are grouped into a custom sequence segment.</p>
                <p>The third step is selecting a comparison receptor set. The selected receptors will be compared to the reference receptor
                based on the selected sequence segments, and their similarities computed. The user can select as many receptors as he/she wishes
                (WARNING: selecting a large number of receptors increases loading time).</p>
                <p>After completing these three steps, an alignment is displayed, with the receptors in the comparison set ranked by similarity
                to the reference receptor. The three columns to the right of the receptor ID show three computed properties:
                <ul>
                    <li><b>Sequence identity (%I)</b>: The percentage of identical amino acids.</li>
                    <li><b>Sequence similarity (%S)</b>: The percentage of similar amino acids (where similar is defined as BLOSUM62 score > 0).</li>
                    <li><b>Similarity score (S)</b>: The sum of every position's BLOSUM62 score.</li>
                </ul>
                To display the sequence number of an aligned residue, as well as generic number indices, hover the mouse over it.
            </section>
            <section>
                <header><h3>Trees</h3></header>
                <p>The "Trees" tool allows for generation of phylogenetic trees based on user selected receptors and sequence segments.</p>
                <p>Using the trees tool is a two step process. The user is first presented with a receptor selection page. Receptors
                can be selected individually or by family. The user can select as many receptors as he/she wishes (WARNING: selecting a
                large number of receptors increases loading time).</p>
                <p>After receptors have been selected, the user is presented with a sequence segment selection page. The user can
                select one or more sequence segments, and/or expand each segment to select the residues within it individually. Residues
                selected individually are grouped into a custom sequence segment.</p>
                <p>In the third step, a settings page is displayed. The amount of bootstrapping replicas (0, 10 or 100) and the type of
                tree (rectangular or circular) are configurable by the user.</p>
                <p>To view an alignment of the sequences used to generate the tree after it has been displayed, click the "View alignment"
                button.</p>
                <p>The trees are generated using <a href='http://evolution.genetics.washington.edu/phylip.html' target='_blank'>PHYLIP</a>
                and <a href='http://www.jsphylosvg.com/' target='_blank'>jsPhyloSVG</a>.</p>
            </section>
            <section>
                <header><h3>Sequence motifs</h3></header>
                <p>The "Sequence motifs" tool allows a user to search a set of receptors for a sequence motif consisting of residue
                positions and chemical properties.</p>
                <p>Using the tools is a two step process. The user is first presented with a receptor selection page. Receptors
                can be selected individually or by family. The user can select as many receptors as he/she wishes (WARNING: selecting a
                large number of receptors increases loading time).</p>
                <p>After receptors have been selected, the user is presented with a sequence motif selecetion page. Motif residues should be
                selected individually. Clicking the down arrow button next to a sequence segment will expand the residues within that segment.
                Chemical features (Hydrophobic, hydrogen bond donor, etc.) should then be selected for each motif residue. When a feature has 
                been selected, a list of amino acids that match the feature will appear to the right of the residue. The list can be edited by
`               clicking on the customise button.</p>
                <p>The selected residues can be organised into separate interactions. An interaction can contain one or more residues. To 
                add an interaction, click the 'Add interaction' button. Selected residues will be added to the currently active interaction
                (shown in bold text). To change the active interaction, click on the name of the interaction. Within an interaction, the
                number of residues required to match can be specified in the 'Min. match' selection box.</p>
                <p>After completing these two steps, an alignment is displayed. The sequences of the selected receptors are split into
                "Matching sequences" and "Non-matching sequences", according to their match of the selected sequence motif. Hovering the
                mouse over the generic number for each residue position will show the list of allowed residues in that position. To display
                the sequence number of an aligned residue, as well as generic number indices, hover the mouse over it.</p>
            </section>
            <section>
                <header><h3>Similarity matrix</h3></header>
                <p>Using the similarity matrix tool is a two step process. The user is first presented with a receptor selection page. Receptors
                can be selected individually or by family. The user can select as many receptors as he/she wishes (WARNING: selecting a
                large number of receptors increases loading time).</p>
                <p>After receptors have been selected, the user is presented with a sequence
                segment selection page. The user can select one or more sequence segments, and/or expand each segment to select the residues
                within it individually. Residues selected individually are grouped into a custom sequence segment.</p>
                <p>After completing these two steps, a matrix is displayed. The sequence similarity of each pair of receptors is displayed in
                the matrix cell where the two intersect. The similarity is the percentage of similar amino acids (where similar is
                defined as BLOSUM62 score > 0).</p>
            </section>
        </article>
    </div>
</div>